applications/osmo4_android_studio/app/src/main/java/com/gpac/Osmo4/GpacConfig.java

```diff
@@ -65,7 +65,10 @@ public GpacConfig(Context context) {
         
         //check if GPAC.cfg exists in /sdcard/osmo
         File gpac_cfg = new File(osmo.getAbsolutePath(), "GPAC.cfg");
-        gpacConfigDirectory = osmo.getAbsolutePath() + "/";
+		if (gpac_cfg.exists())
+			gpacConfigDirectory = osmo.getAbsolutePath() + "/";
+		else
+			gpacConfigDirectory = null;
     }
```

Comment:
1) this reverts 570873d75bfa46999294cf3d5c6b4be3b227df1d
2) there an indent issue